nerfstudio/scripts/downloads/download_data.py

```diff
@@ -555,22 +542,15 @@ def download(self, save_dir: Path) -> None:
     Annotated[SDFstudioDemoDownload, tyro.conf.subcommand(name="sdfstudio")],
     Annotated[NeRFOSRDownload, tyro.conf.subcommand(name="nerfosr")],
     Annotated[Mill19Download, tyro.conf.subcommand(name="mill19")],
+    Annotated[EyefulTowerDownload, tyro.conf.subcommand(name="eyefultower")],
 ]
 
 
 def main(
     dataset: DatasetDownload,
 ):
     """Script to download existing datasets.
-    We currently support the following datasets:
-    - nerfstudio: Growing collection of real-world scenes. Use the `capture_name` argument to specify
-        which capture to download.
-    - blender: Blender synthetic scenes realeased with NeRF.
-    - sitcoms3d: Friends TV show scenes.
-    - record3d: Record3d dataset.
-    - dnerf: D-NeRF dataset.
-    - phototourism: PhotoTourism dataset. Use the `capture_name` argument to specify which capture to download.
-    - mill19: Mill 19 dataset. Use the `capture_name` argument to specify which capture to download.
+    We currently support the datasets listed above in the Commands.
```

Comment:
Instead of deleting the comment, can you keep it and add eyefultower to the list with a description? Also remove the "above in the Commands." syntax and keep it what it was, "We currently support the following datasets:" because this will be printed in the terminal and people won't see anything "above" unless digging into the code.

Reply:
I deleted the entire section because it was out of date, missing a few of the more recent additions (SDFstudio, NeRFOSR), and felt a little redundant. In code, you can see the list of supported datasets in the `Commands` variable a couple lines up. On the command line, when running `--help`, you can see the list of supported datasets as the subcommands. I don't actually see this block printed anywhere in the command line - does it show up in a different command somewhere?

```
(nerfstudio12) PS C:\Users\vasuagrawal\Documents\nerfstudio> python .\nerfstudio\scripts\downloads\download_data.py --help
usage: download_data.py [-h] {blender,sitcoms3d,nerfstudio,record3d,dnerf,phototourism,sdfstudio,nerfosr,mill19,eyefultower}

╭─ arguments ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ -h, --help        show this help message and exit                                                                                                                                                                                                                                                                                               │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
╭─ subcommands ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ {blender,sitcoms3d,nerfstudio,record3d,dnerf,phototourism,sdfstudio,nerfosr,mill19,eyefultower}                                                                                                                                                                                                                                                 │
│     blender       Download the blender dataset.                                                                                                                                                                                                                                                                                                 │
│     sitcoms3d     Download the sitcoms3D dataset.                                                                                                                                                                                                                                                                                               │
│     nerfstudio    Download data in the Nerfstudio format. If you are interested in the Nerfstudio Dataset subset from the SIGGRAPH 2023 paper, you can obtain that by using --capture-name nerfstudio-dataset or by visiting Google Drive directly at: https://drive.google.com/drive/folders/19TV6kdVGcmg3cGZ1bNIUnBBMD-iQjRbG?usp=drive_link. │
│     record3d      Download the record3d dataset.                                                                                                                                                                                                                                                                                                │
│     dnerf         Download the dnerf dataset.                                                                                                                                                                                                                                                                                                   │
│     phototourism  Download the phototourism dataset.                                                                                                                                                                                                                                                                                            │
│     sdfstudio     Download the sdfstudio dataset.                                                                                                                                                                                                                                                                                               │
│     nerfosr       Download the NeRF-OSR dataset.                                                                                                                                                                                                                                                                                                │
│     mill19        Download the Mill 19 dataset.                                                                                                                                                                                                                                                                                                 │
│     eyefultower   Download the EyefulTower dataset. Use the --help flag with the `eyefultower` subcommand to see all available datasets. Find more information about the dataset at https://github.com/facebookresearch/EyefulTower.                                                                                                            │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
(nerfstudio12) PS C:\Users\vasuagrawal\Documents\nerfstudio> python .\nerfstudio\scripts\downloads\download_data.py
╭─ Required arguments ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ The following arguments are required: {blender,sitcoms3d,nerfstudio,record3d,dnerf,phototourism,sdfstudio,nerfosr,mill19,eyefultower} │
│ ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── │
│ For full helptext, run download_data.py --help                                                                                        │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

Reply:
Gotcha. I'm not actually sure why that doesn't show up, but the `--help` printout looks great. Thanks for explaining this update.

Reply:
Yeah, this actually won't be printed in the terminal because we're effectively running:
```python
command = tyro.cli(Commands)
main(command)
```
where `main` itself is never passed to the CLI generator, so the docstring won't be used in any helptext.

The docstring _would_ be used if we instead ran something like `tyro.cli(main)` (which would basically also work)